subsys/suit/provisioning/CMakeLists.txt

```diff
@@ -191,14 +191,18 @@ function (generate_mpi_area area)
   message(INFO " Generate merged MPI for ${area} (${output})")
 endfunction()
 
-if(DEFINED CONFIG_SOC_SERIES_NRF54HX)
-configure_storage_address_cache()
-endif() # CONFIG_SOC_SERIES_NRF54HX
+if(DEFINED CONFIG_SOC_SERIES_NRF54HX OR CONFIG_SOC_SERIES_NRF92X)
+  configure_storage_address_cache()
+endif() # CONFIG_SOC_SERIES_NRF54HX OR CONFIG_SOC_SERIES_NRF92X
 
 if(DEFINED CONFIG_SUIT_MPI_GENERATE)
 
-if(DEFINED CONFIG_SUIT_MPI_SOC_NRF54H20)
-include(soc/nrf54h20.cmake)
-endif() # CONFIG_SUIT_MPI_SOC_NRF54H20
+  if(DEFINED CONFIG_SUIT_MPI_SOC_NRF54H20)
+    include(soc/nrf54h20.cmake)
```

Comment:
FYI: MPIs are (finally!) moved to the cmake/sysbuild: https://github.com/nrfconnect/sdk-nrf/pull/19016

Reply:
Updated.